### ISSUE TITLE
Cannot get popup footnotes/endnotes working!

### ISSUE THREAD

**Issue by jeff47** [kobolabs/epub-spec#64]
I am at a loss here, trying to get footnotes & endnotes working.

As far as I can see, I am using the correct syntax, copied from the epub-spec here.  When I open the epub in Books or Calibre on my desktop, I get popup footnotes and endnotes exactly as expected.  After conversion to kepub and moved to my Libra H2O, they do not.  I copied the syntax from an example that did work on my Libra H2O, and compared both the ePub and kepub formats.  I don't see significant differences, and feel like I'm losing my mind a little.

This zip contains the epub and the converted kepub I've been trying to get working:  [Footnote tests.zip](https://github.com/kobolabs/epub-spec/files/12209881/Footnote.tests.zip)

If it is easier, here are direct links to the version that doesn't work:
- https://github.com/jeff47/kobo-footnote-tests/blob/main/Footnotes.epub
- https://github.com/jeff47/kobo-footnote-tests/blob/main/Footnotes.kepub

**Comment by mramosch** [comment]
I have them both working on a Clara BW.

Just being a bit confused about your naming things, like file name says version 2 but the actual content says version 3 but you do actually have two pairs of text/endnotes which is 4 items and both pairs are called version 3.

Which exactly are the ones working, which don’t? 

Before I open them up can you clarify please…

One thing I saw right away was the file extension of the second file which is supposed to be ‘name.kepub.epub’.

Let me know whether you have already resolved your problems.

**Comment by DanteDT** [comment]
> I have them both working on a Clara BW, in general, not pop-ups though…
> 
> A remedy I found was to NOT distribute the footnote attributes over two elements.
> 
> When shifting the ‘aside’ attributes over to the element with the back-link, both directions do work, even on the epub file ...
> 

Yes to @mramosch - this is a great find "to NOT distribute the footnote attributes over two elements."

Unfortunately, that other part remains: "not pop-ups though..."

Kobo continues not to reliably support a great reader experience with popup notes.
I can see that others are facing this frustration unaddressed for years. Like @jeff47, I've now lost a little bit of my mind over this non-conformance, which is disappointing.
